Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/AzureModule.java

```diff
@@ -122,8 +124,13 @@ private static String composeName() {
             return String.format("%s (%s)", BASE_MODULE_NAME, getAccountDescription(selectedSubscriptions));
 
         } catch (Exception e) {
-            final String msg = String.format(ERROR_GETTING_SUBSCRIPTIONS_MESSAGE, e.getMessage());
-            DefaultLoader.getUIHelper().showException(msg, e, ERROR_GETTING_SUBSCRIPTIONS_TITLE, false, true);
+            if (e instanceof AzureRuntimeException) {
+                DefaultLoader.getUIHelper().showInfoNotification(ERROR_GETTING_SUBSCRIPTIONS_TITLE
+                        , ErrorUIMapEnum.getViewMessageByCode(((AzureRuntimeException) e).getCode()));
```

Comment:
@andxu , is comma ahead allowed in coding style?

Reply:
updated.